test/integration/vinyldns_test.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2019 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const assert = require('assert');
+const VinylDns = require('../../src/vinyldns');
+
+const vinyl = new VinylDns({
+  apiUrl: 'http://localhost:9000',
+  accessKeyId: 'okAccessKey',
+  secretAccessKey: 'okSecretKey'
+});
+
+function group() {
+  return {
+    name: 'ok-group',
+    description: 'description',
+    email: 'test@test.com',
+    members: [{
+      userName: 'ok',
+      id: 'ok'
+    }],
+    admins: [{
+      userName: 'ok',
+      id: 'ok'
+    }]
+  };
+}
+
+function zone(groupId) {
+  return {
+    adminGroupId: groupId,
+    name: 'vinyldns.',
+    email: 'test@example.com'
+  };
+}
+
+describe('VinylDns interaction with a real VinylDNS API', () => {
```

Comment:
Maybe change `VinylDns` → `VinylDNS` here for consistency.

Reply:
Sounds good - I like that.